src/config.cpp

```diff
@@ -784,6 +784,17 @@ const config::attribute_value* config::get(config_key_type key) const
 	return i != values_.end() ? &i->second : nullptr;
 }
 
+/**
+ * Chooses a value. If the value specified by @a key is
+ * blank, then @a default_key is chosen instead.
+ */
```

Comment:
Duplicated documentation - for this class, please put it in the header file only.